ehrql/tables/beta/tpp.py

```diff
@@ -459,3 +460,48 @@ class isaric_raw(EventFrame):
         datetime.date,
         description="Outcome date.",
     )
+
+
+@table
+class open_prompt(EventFrame):
+    """
+    Participants answer questions in questionnaires. Their answers are responses. There
+    is one questionnaire that we expect to be administered once, on day 0. There are
+    five questionnaires that we expect to be administered four times, on days 0, 30, 60,
+    and 90.
+
+    * Baseline questionnaire (administered once, on day 0)
+    * Covid and vaccination questionnaire
+    * Eq-5D questionnaire
+    * Productivity questionnaire
+    * Fatigue questionnaire
+    * Breathing problems questionnaire
+
+    Each time is a survey. That is, a survey comprises the responses to the questions in
+    between five and six questionnaires.
+    """
+
+    coded_event_id = Series(
```

Comment:
This is an odd one because the column name TPP have given it is very much at odds with what it actually seems to contain. If our understanding of what it does contain is correct I'd be inclined to call it `ctv3_code`, but the disparity makes me a bit nervous.

Reply:
Yes, me too.

The values in `CodedEvent_ID`[^1] match (by eye) the values in the _CTV3 codes_ column of [the codelist that LSHTM provided][1], which is why I assumed they were CTV3 codes. This could be a mis-labelling on TPP's part or on LSHTM's part. I'm leaning towards the latter.

The `ConceptId` column contains CTV3 codes. I updated the property name to reflect this in 8b4e506. So, I'll not replace `coded_event_id` with `ctv3_code` just yet.

[1]: https://docs.google.com/spreadsheets/d/1bmiQH2Y4EH_PQ4ecaadq_0jWvTHnK0s9/edit#gid=155161282

[^1]: See <https://github.com/opensafely/airmid-short-data-report> and <https://jobs.opensafely.org/datalab/opensafely-internal/airmid-short-data-report/>

Reply:
See above. We now have `ctv3_code` and `snomedct_code`.